deployment/azuredeploy.json

```diff
@@ -470,7 +469,7 @@
                 "AzCliVersion": "2.0.80",
                 "timeout": "PT30M",
                 "arguments": "[variables('cliArg')]",
-                "scriptContent": "az keyvault key create --vault-name $1 --name $5 --protection software;az storage blob service-properties update --account-name $2 --static-website --404-document index.html --index-document index.html;",
+                "scriptContent": "az account list",

```

Comment:
I would like to remove this step altogether. It is doing nothing at the moment except confusing the reader about its existence. I can understand the urgency of getting this fix in since the current state is a customer impact, however, I'd request you to please follow-up with an immediate PR that has the cleaned up implementation of this change.